tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/compiler/N4jscTestCatalogIgnoresTestsInNodeModules.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ide.tests.compiler;
+
+import static org.eclipse.n4js.cli.N4jscExitCode.VALIDATION_ERRORS;
+import static org.eclipse.n4js.cli.N4jscTestOptions.COMPILE;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.n4js.N4JSGlobals;
+import org.eclipse.n4js.cli.helper.AbstractCliCompileTest;
+import org.eclipse.n4js.cli.helper.CliCompileResult;
+import org.eclipse.n4js.utils.io.FileDeleter;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests that the test catalog file does not contain tests of packages located inside the node_modules folder.
+ */
+public class N4jscTestCatalogIgnoresTestsInNodeModules extends AbstractCliCompileTest {
```

Comment:
Must follow naming convention for tests (to be executed in maven builds).

Reply:
👍  Thanks!